modules/Bio/EnsEMBL/Variation/DBSQL/VariationFeatureAdaptor.pm

```diff
@@ -1761,6 +1761,17 @@ sub fetch_by_hgvs_notation {
   throw ("Could not parse the HGVS notation $hgvs") 
       unless (defined($reference) && defined($type) && defined($description));
 
+  # ClinVar notations we don't support:
+  # 88957372-?_89042944+?dup
+  # (?145413072)(145592772_145596903)del
+  if($description =~ m/(\-\?\_.+\?)|(\(\?\_.+\))/) {
+    throw ("HGVS notation is not supported");
```

Comment:
It would be clearer to say 'HGVS notation for variation with unknown locations is not supported'

Reply:
Thanks for the suggestion. This way is more specific.